views/movies/movie-details.hbs

```diff
@@ -0,0 +1,16 @@
+<h2>Movie Details</h2>
+
+
+<h3></h3>{{this.movieDetails.title}}</h3>
+<p>Genre: {{this.movieDetails.genre}}</p>
+<p>Plot: {{this.movieDetails.plot}}</p>
+
+{{#each movieDetails}}
+<p>{{this.cast}}</p>
+<p>{{this.cast.occupation}}</p>
+<p>{{this.cast.catchPhrase}}</p>
+{{/each}}
```

Comment:
You are already inside movie details. We are looking at one single movie at this point -> so you want to iterate over the `movieDetails.cast `